src/json_parser.hpp

```diff
@@ -13,7 +13,7 @@ enum class JsonValueType { Null, Boolean, Number, String, Array, Object };
 struct JsonValue {
     JsonValueType type;
     std::string string_value;
-    double number_value;
+    long double number_value;
```

Comment:
I think that's ok for our purposes but there is no guarantee that "long double" can represent all `uint64_t`. 

But that's ok.

Reply:
Fair point, added a small comment as a note on precision.